src/bezpath.rs

```diff
@@ -1535,4 +1592,65 @@ mod tests {
             .collect::<Vec<_>>();
         assert_eq!(segments, get_segs);
     }
+
+    #[test]
+    fn test_control_box() {
```

Comment:
Too much attention to detail :) I added control_box earlier and missed the test. Since I was copying tests over for contour reversal, I thought I might sneak this in.

Caught!